templates/nginx_stage.yml.erb

```
@@ -28,7 +28,9 @@ ondemand_portal: '<%= scope['openondemand::nginx_stage_ondemand_portal'] -%>'
 # NB: If this is not set then most apps will use default title "Open OnDemand"
 #
 #ondemand_title: null
+<% if scope['openondemand::nginx_stage_ondemand_title'] -%>
 ondemand_title: '<%= scope['openondemand::nginx_stage_ondemand_title'] -%>'
```

```suggestion
ondemand_title: '<%= scope['openondemand::nginx_stage_ondemand_title'] %>'
```
One minor thing.  In this particular context it won't hurt but stripping the trailing white space in ERB when using `<%=` can sometimes have really weird results.  Also this conforms to other ERB printing in this file.